.travis.yml

```diff
@@ -3,12 +3,14 @@ node_js:
   - "6"
   - "8"
   - "10"
+  - "12"
```

Comment:
Please remove build changes from this PR. I'll consider updating this separately.

Reply:
Updated this already: db1b2c1dc9ed11035ff5ae4b28515bbf891935dd